### DISCUSSION TITLE
Twinning Boundaries

### DISCUSSION THREAD

**Discussion by WilliamsB39** [mtex-toolbox/mtex#1480] [Ask Anything]
Hi all,
I am trying to highlight twin boundaries of Anorthite by following the procedure in: https://mtex-toolbox.github.io/TwinningBoundaries.html. However, I have issues with defining the twinning.

I am able to follow the workflow until the point where the mori_mean and round2Miller are shown:

  Bunge Euler angles in degree
     phi1     Phi    phi2    Inv.
  356.141  179.84 181.298       0
 
plane parallel   direction parallel   fit
(010) || (010)   [00-1] || [001]     0.661°

I followed the answers in this discussion https://github.com/mtex-toolbox/mtex/discussions/975 and defined the twinning based on the above as:
```
twinning = orientation.map(Miller({0 1 0},CS), Miller({0 1 0},CS),...
    Miller({0 0 -1},CS), Miller({0 0 1},CS));
```
But I am getting these errors:
```
twinning = orientation.map(Miller({0 1 0},CS2), Miller({0 1 0},CS2),...
    Miller({0 0 -1},CS2), Miller({0 0 1},CS2));
Error using rotation.map (line 35)
Inconsitent pairs of vectors! The angle between u1, u2 and v1, v2 needs to be the same, but differs by 8.1718°

Error in orientation.map (line 29)
ori = orientation(rotation.map(varargin{:}));
```

Does anyone have any idea what I am doing wrong/need to do?
Any help would be greatly received.
Thanks,
Ben

**Comment by kilir** [comment]
Hi,
don't for get your phase is triclinic! the definition you want to use is (hkl) || (hkl) and [uvw] || [uvw].

```
CS = loadCIF('9001258')

twinning = orientation.map(Miller(0, 1, 0,CS), Miller(0, 1, 0,CS),...
    Miller(0, 0, -1,CS,'uvw'), Miller(0, 0, 1,CS,'uvw'));
```

Cheers,
Rüdiger

**Reply by WilliamsB39** [reply]
Yes, of course. Thanks for your help!
Ben
